pkg/security/api/validation/validation_test.go

```diff
@@ -23,12 +30,13 @@ func validPodSpec() kapi.PodSpec {
 		DNSPolicy:             kapi.DNSClusterFirst,
 		ActiveDeadlineSeconds: &activeDeadlineSeconds,
 		ServiceAccountName:    "acct",
+		SchedulerName:         kapi.DefaultSchedulerName,
 	}
 }
 
 func invalidPodSpec() kapi.PodSpec {
 	return kapi.PodSpec{
-		Containers:    []kapi.Container{{}},
+		Containers:    []kapi.Container{{TerminationMessagePolicy: kapi.TerminationMessageReadFile}},
```

Comment:
so is ImagePullPolicy, but that's not specified here, so I wouldn't expect this to actually be getting defaulted